docs/openapi/components/schemas/common/CBPEntrySummary.yml

```diff
@@ -199,10 +199,10 @@ properties:
   importerNumber:
     title: Importer Identifier
     description: Record the IRS EIN, SSN, or CBP assigned number of the importer of record. Proper format is listed under the instructions for Consignee Number.
-    type: string
+    $ref: ./CBPImporterOfRecord.yml
     $linkedData:
       term: importerNumber
-      '@id': https://schema.org/identifier
+      '@id': https://w3id.org/traceability#importerOfRecord
```

Comment:
sad that we define this

Reply:
Good point. 

How about `Declarant's Customs identity number`: https://vocabulary.uncefact.org/ReferenceCodeList#ABP